### ISSUE TITLE
Compatability with Elasticsearch 2.0 

### ISSUE THREAD

**Issue by imranazad** [yakaz/elasticsearch-analysis-combo#22]
This plugin doesn't install with Elasticsearch 2.0, I get the following error: ERROR: Could not find plugin descriptor 'plugin-descriptor.properties'

**Comment by Kijewski** [comment]
→ https://github.com/yakaz/elasticsearch-analysis-combo/pull/21

**Comment by imranazad** [comment]
I tried running the following command: 

> plugin.bat install "file:///elasticsearch-2.0.0/plugins/elasticsearch-analysis-combo-es20.zip" 

but I'm still getting an error regarding  'plugin-descriptor.properties' even though I can see it exists.

**Comment by Kijewski** [comment]
Yeah, but you should comment on the existing PR instead of opening a new issue, or the discussion will become cluttered. @antonha would probably be happy about your input.

**Comment by imranazad** [comment]
Ah my bad, OK, thanks.
